odoo/addons/test_new_api/tests/test_properties.py

```diff
@@ -0,0 +1,1064 @@
+# -*- coding: utf-8 -*-
+import json
+
+from unittest.mock import patch
+
+from odoo import Command
+from odoo.exceptions import AccessError, CacheMiss
+from odoo.tests.common import Form, TransactionCase
+from odoo.tools import mute_logger
+
+
+class PropertiesCase(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.user = cls.env['res.users'].search([], limit=1)
+        cls.partner = cls.env['res.partner'].search([], limit=1)
+
+        message_properties_1 = [
+            {
+                'id': 'name',
```

Comment:
Would avoid using "name" and "parent_id" as all models tend to have that field already.

How about something a bit more fancy, like "discussion_color_code" and "moderator_partner_id" or something like that?